planner/languages/rust/rust_planner.go

```diff
@@ -4,9 +4,16 @@
 package rust
 
 import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.jetpack.io/devbox/cuecfg"
 	"go.jetpack.io/devbox/planner/plansdk"
 )
 
+const cargoToml = "Cargo.toml"
```

Comment:
Is it always capitalized?

Reply:
I thought it was https://github.com/rust-lang/cargo/issues/45
but when I tried `cargo build` on a `cargo.toml` project, it did work, so updated code to consider both.

Reply:
Actually, `cargo` does complain about `cargo.toml`. If I do:
```
 > [builder 6/6] RUN --mount=type=cache,target=/nix/store,from=base,source=/nix/store cargo build --release:
#28 0.323 + cargo build --release
#28 0.343 error: could not find `Cargo.toml` in `/scratch` or any parent directory, but found cargo.toml please try to rename it to Cargo.toml
```

I think our code can support it since `cargo build` works, but an FYI